elixir/plan.sh

```diff
@@ -18,5 +19,7 @@ do_prepare() {
 
 do_build() {
     fix_interpreter "rebar" core/coreutils bin/env
+    fix_interpreter "rebar3" core/coreutils bin/env
```

Comment:
Theoretically the `rebar3` interpreter can be ignored. The `rebar` binary at the root of the source is needed to build Elixir, so that can't be changed.